storage/ndb/include/ndbapi/NdbOperation.hpp

```diff
@@ -1131,7 +1131,9 @@ class NdbOperation {
       OO_TTL_IGNORE    = 0x8000,
       OO_INTERPRETED_INSERT = 0x10000,
       OO_DIRTY_FLAG = 0x20000,
-      OO_TTL_ONLY_EXPIRED = 0x40000
+      OO_TTL_ONLY_EXPIRED = 0x40000,
+      OO_BATCH_SAFE_FLAG = 0x80000,
+      OO_BATCH_UNSAFE_FLAG = 0x100000
```

Comment:
Why do you need two flags? Shouldn't we assume batching is unsafe if safe flag is 0?

Reply:
Nope, safe gives us the ability to use batching when deemed unsafe, unsafe means it is unsafe to batch even when considered safe

Reply:
Batching or not is controlled by several variables and conditions